resources/assets/js/components/EventTitle.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <component :is="component">
```

Comment:
Ah this interesting, it's a way to make a component template be configured to use different html elements?

Reply:
Yes, exactly - not just native HTML elements, though, it works with different Vue components.